test/clj_kondo/impl/analyzer_test.clj

```diff
@@ -109,7 +112,48 @@
              (do
                (reset! findings [])
                (analyze "1..1")
-               @findings))))))
+               @findings))))
+
+    (testing "duplicate map keys"
```

Comment:
This test should be added to clj_kondo/main_test.clj: `deftest map-duplicate-keys`. This file should not be touched for this feature.

Reply:
Addressed this in the latest commits.